pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -67,25 +68,29 @@ in stdenv.mkDerivation rec {
   '';
 
   cmakeFlags = [
-    "-DARROW_BUILD_TESTS=ON"
+    "-DARROW_BUILD_TESTS=${if static then "OFF" else "ON"}"
```

Comment:
I think it’s a bad idea to run arrow without tests. Why can’t those stay enabled?

Reply:
I just disabled options until it worked at first. I updated the commit and this is no longer an issue.